app/templates/application.hbs

```diff
@@ -4,13 +4,10 @@
 
 <header local-class="header">
   <LinkTo @route="index" @tabindex="-1">
-    <img src="/assets/Cargo-Logo-Small.png" height="100" width="100" alt="Cargo Logo">
+    <img src="/assets/Cargo-Logo-Small.png" alt="Cargo Logo" local-class="logo">
```

Comment:
That alt text is not very helpful. I recommend adding `role="presentation"` and use `alt=""` as it does not hold any actual content value except for being a decorative addition to the name.

I also recommend moving the image inside the second link, dropping the duplicate.